src/utils.js

```diff
@@ -38,6 +38,15 @@ const setupCml = async opts => {
       version !== 'latest' ? `@${version}` : ''
     }`
   );
+
+  let installedVersion = '';
+  try {
+    const json = await exec('npm list --json --global @dvcorg/cml');
+    installedVersion = JSON.parse(json).dependencies['@dvcorg/cml'].version;
+    console.log(
+      `Targeted CML version: ${version}, received: ${installedVersion}`
+    );
+  } catch (err) {}
```

Comment:
If we aren't able to determinte the installed version, wouldn't it be better to let the error reach the user?

Reply:
I agree and admit I was lazy and followed the pattern from established from attempting to grab the sudoPath above, should both be spots resolved here?